deepchem/models/tensorgraph/tensor_graph.py

```diff
@@ -765,16 +765,30 @@ def save_checkpoint(self, max_checkpoints_to_keep=5):
     saver = tf.train.Saver(max_to_keep=max_checkpoints_to_keep)
     saver.save(self.session, self.save_file, global_step=self.global_step)
 
-  def restore(self):
-    """Reload the values of all variables from the most recent checkpoint file."""
+  def get_checkpoints(self):
+    """Get a list of all available checkpoint files."""
+    return tf.train.get_checkpoint_state(
+        self.model_dir).all_model_checkpoint_paths
+
+  def restore(self, checkpoint):
```

Comment:
Should `checkpoint` be a kwarg `checkpoint=None`?

Reply:
Yes, thanks.  I just fixed it.